### DISCUSSION TITLE
Good first issues ?

### DISCUSSION THREAD

**Discussion by mcsky** [rectorphp/rector#8401] [General]
Hello ! 

You did a great library, thanks for your work !

I would like to improve my knowledge of the AST, so I propose to help you if you need to. 
Do you have a list of "good start issue", that I could try to implement ?

**Comment by TomasVotruba** [comment]
Hi,

thank you for joining!

Here is one to kick it off:
https://github.com/rectorphp/rector/issues/8384

**Reply by mcsky** [reply]
Just to be sure that I'm not going off road. 

I only need to replace the FQCN `Symfony\Component\Routing\Annotation\Route` to `Symfony\Component\Routing\Attribute\Route` ?

Or should I also replace the @Route definition by the equivalent #[Route] ?

**Reply by samsonasik** [reply]
I think you can just register `RenameClassRector` to rename `Symfony\Component\Routing\Annotation\Route` to `Symfony\Component\Routing\Attribute\Route` is enough after `AnnotationToAttributeRector` definition, so existing attribute if already exist will be renamed.

see https://github.com/rectorphp/rector-symfony/blob/c9d78b386a7658e3f9ad71ad055a38eca436ee69/config/sets/symfony/annotations-to-attributes.php#L17

**Reply by mcsky** [reply]
Thanks for your tips @samsonasik , I'm pretty new on rector so I'm still looking for the big picture. I gave it a try in https://github.com/rectorphp/rector-symfony/pull/557 :)
